src/hooks/useError.js

```diff
@@ -0,0 +1,16 @@
+import { useState, useCallback } from 'react';
+
+const useAsyncError = () => {
+  // eslint-disable-next-line no-unused-vars
+  const [_, setError] = useState();
```

Comment:
Not sure what the _ means?

Reply:
That's just a convention when you don't care about the first parameter. javascript does not allow you to do `const [, setError] = useState();` so `_` usually signifies an unused param.